onnx_xla/onnx_xla_interface.cc

```diff
@@ -21,6 +24,13 @@ struct OnnxXlaBackendID {
   int device_id{0};
 };
 
+struct EventControl {
+  EventControl() : signalled(false) {}
+  bool signalled;
+  std::mutex m;
```

Comment:
Please use more descriptive names, i.e. `mutex` and e.g. `condvar`. `e->m` in the code looks cryptic.

Reply:
Changed to signalled_, mutex_, and condvar_. Also change e to eventController